<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>168</width>
    <height>488</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>168</width>
    <height>488</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>168</width>
    <height>488</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Amount</string>
  </property>
  <widget class="QDialogButtonBox" name="button_amount">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>151</width>
     <height>401</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>151</width>
     <height>401</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTableWidget" name="amount_atack">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>385</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>385</height>
     </size>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>63</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>63</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>Spear</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Sword</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Axe</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Archer</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Spy</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Light</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Marcher</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Heavy</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Ram</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Catapult</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Knight</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Snob</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Amounts</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_amount</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_amount</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
